crates/transpiler/src/target/mod.rs

```diff
@@ -1538,8 +1533,8 @@ impl Target {
         let num_bounds = bounds.len();
         let operation = self.operation_from_name(name);
         let num_params = match operation {
-            Some(op) => {
-                let params = op.params();
+            Some(TargetOperation::Normal(op)) => {
+                let params = op.params_view();
```

Comment:
This, and the next comment, suggest a behavioural change, and causes an incorrect error message - we now send CF ops down the path that causes an error message that says ``"{name} is not an instruction in the `Target`"``, even if it is.

Reply:
Fixed in 5ffb2115b8.